### PR TITLE
Staging to prod, MicroStream post

### PR DESCRIPTION
**Pull request by dmuelle**
#2227

### CODE REVIEW

File: posts/2022-05-03-microstream-cdi-ext.adoc [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+[source,Java]
+----
+	@Store(root = true)
+----
+
+The code of this example is also available in the https://github.com/microstream-one/microstream/tree/master/examples/openliberty[MicroStream repository].
+
+== Conclusion
+
+With the MicroStream framework, you can quickly and efficiently query and manipulate the application data very as regular Java class instances that don't involve any mapping. You can also store the data to any kind of blob storage in a safe way that doesn't suffer the Java serialization security vulnerabilities.
```
Comment (mbroz2) [inline posts/2022-05-03-microstream-cdi-ext.adoc]:
```suggestion
With the MicroStream framework, you can quickly and efficiently query and manipulate the application data as regular Java class instances that don't involve any mapping. You can also store the data to any kind of blob storage in a safe way that doesn't suffer the Java serialization security vulnerabilities.
```

File: posts/2022-05-03-microstream-cdi-ext.adoc [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+layout: post
+title: "Easy and fast data access in Open Liberty with the MicroStream CDI extension"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/rdebusscher
+author_github: https://github.com/rdebusscher
+seo-title: Easy and fast data access in Open Liberty with the MicroStream CDI extension - OpenLiberty.io
+seo-description: The MicroStream CDI extension can be used to integrate the MicroStream framework into a MicroProfile application that runs on Open Liberty. It enables you to store Java object graphs natively and securely, with no heavyweight DBMS dependencies.
+blog_description: "The MicroStream CDI extension can be used to integrate the MicroStream framework into a MicroProfile application that runs on Open Liberty. It enables you to store Java object graphs natively and securely, with no heavyweight DBMS dependencies."
+open-graph-image: https://openliberty.io/img/blog/blog_microstream_starter.png
```
Comment (mbroz2) [inline posts/2022-05-03-microstream-cdi-ext.adoc:11]: This image, used here and in the blog body, doesn't seem to exist in staging nor prod.

**Review by mbroz2 — CHANGES_REQUESTED** [review]

File: posts/2022-05-03-microstream-cdi-ext.adoc (hunk 2 above)
Comment (dmuelle) [reply posts/2022-05-03-microstream-cdi-ext.adoc:11]: added image

File: posts/2022-05-03-microstream-cdi-ext.adoc (hunk 1 above)
Comment (dmuelle) [reply posts/2022-05-03-microstream-cdi-ext.adoc]: fixed

**Review by mbroz2 — APPROVED** [review]

**Pull request merged**